src/main/kotlin/com/openlattice/organization/DatasetApi.kt

```diff
@@ -9,9 +11,11 @@ const val CONTROLLER = "/organization-database"
 const val BASE = SERVICE + CONTROLLER
 
 const val EXTERNAL_DATABASE = "/external-database"
-const val EXTERNAL_DATABASE_COLUMN = "/external-database-column"
-const val EXTERNAL_DATABASE_TABLE = "/external-database-table"
+const val EXTERNAL_DATABASE_COLUMN = "/column"
```

Comment:
Really minor thing, but can you please also update the names of the constants? Also, this is not used yet in any of the apps right? (path changes might break things)

Reply:
Um I think hristo is currently working on adding this to the orgs app, not sure where we're at with it. I can leave them as they were though if it is safer.